ultraplot/axes/geo.py

```diff
@@ -1673,3 +1768,71 @@ def _update_minor_gridlines(self, longrid=None, latgrid=None, nsteps=None):
 # Apply signature obfuscation after storing previous signature
 GeoAxes._format_signatures[GeoAxes] = inspect.signature(GeoAxes.format)
 GeoAxes.format = docstring._obfuscate_kwargs(GeoAxes.format)
+
+
+def _is_rectilinear_projection(ax):
```

Comment:
There is some missing coverage in this file. We need to add tests to cover branches.

Reply:
I updated the logic to better reflect the logic within the class. Some of the checks were a bit overkill as they are caught by other functions already. This should satisfy the codecov patches (will see when its updated).  The major change is that some of the checks were a bit overkill since the format function itself sets the projection class and as such we could not check for that with the functions available in pytest, although we could  add tests for private functions (but that would be a bit overkill again).